CONTRIBUTING.md

```diff
@@ -309,6 +309,25 @@ about:
     when someone else is asking for feedback on their code, and universe balances out.
 
 
+Backporting
+-----------
+
+After the original PR was merged into master, backporting is done only
+when necessary - the fix either needs to be present in a release branch
+(e.g. a security issue) or on request.
+If the backport is trivial and can be cherry-picked automatically,
+just request a "Needs backport" tag for the original PR.
+If backport needs any conflict resolution, create the PR manually and
```

Comment:
Could mention that the PR must be for a specific branch, not master. Also, a PR may be needed for each version where the backport makes sense.

nit, drop "wait for comments"?

Reply:
Why a specific branch, not master? I understand that my development is done on a specific branch in my forked repo, but I ask for the Pull Request to master.

"wait for comments" dropped in 1d9ffdd2432f9a89f77204505b221616cf74b704 and will be squashed in the end

Reply:
Backports are done when something is already in master (that is the practice of Linux development, where git itself comes from; in contrary to what Android or other BSP developers tend to do - they add specific hardware patches to a stable branch, making master run away quickly and then they are unable to catch up so after the device is not sold anymore, one can hardly find a current kernel and Android build for it...)

Reply:
> Why a specific branch, not master? I understand that my development is done on a specific branch in my forked repo, but I ask for the Pull Request to master.

Commits are backported *from* the master branch *to* a specific branch, e.g., `0.18` or `0.19`.

Reply:
@promag @hebasto OK, sorry. I totally misunderstood.

Reply:
@promag What do you think about 2aa9503ea ?